hlsp.go

```diff
@@ -1,6 +1,30 @@
-package hlsp
+package parser
+
+import "fmt"
 
 // Parse receives either a YAML or JSON AsyncAPI document. It parses the document and checks if it's valid AsyncAPI. Skips specification extensions and schemas validation. If validation fails, the Parser/Validator should trigger an error. Produces a beautified version of the document in JSON Schema Draft 07
-func Parse() string {
-	return "Hello World"
+func Parse(document string) (string, err error) {
+
+	return
+}
+
+func convertToJSON(doc string) (string, error) {
+	if isJSON(doc) {
+		// convert here JSON to YML
+		return "", nil
+	}
+
+	if isYML(doc) {
```

Comment:
Please can we use YAML/yaml everywhere not YML/yml, which doesn't really mean anything unless you're using DOS or Windows 3.